test/620_proxy_entrypoint_inheritance_test.sh

```diff
@@ -0,0 +1,19 @@
+#! /bin/bash
+
+. ./config.sh
+
+start_suite "Proxy uses entrypoint from the container with weavewait"
+# Setup and sanity-check
+weave_on $HOST1 launch-proxy
+docker_on $HOST1 build -t inspect-ethwe - <<- EOF
+  FROM gliderlabs/alpine
+  ENTRYPOINT ["ip", "link", "show", "ethwe"]
+EOF
+assert "docker_on $HOST1 inspect --format='{{.Config.Entrypoint}}' inspect-ethwe" "[ip link show ethwe]"
+assert "docker_on $HOST1 inspect --format='{{.Config.Cmd}}' inspect-ethwe" "<no value>"
+
+# Boot a new container with no entrypoint of its own, just a command
+assert_raises "docker_proxy_on $HOST1 run -e 'WEAVE_CIDR=10.2.1.1/24' --name=c2 inspect-ethwe" 0
+assert "docker_on $HOST1 inspect --format='{{.Config.Entrypoint}}' c2" "[/home/weavewait/weavewait]"
+assert "docker_on $HOST1 inspect --format='{{.Config.Cmd}}' c2" "[ip link show ethwe]"
+end_suite
```

Comment:
This whole test is too low-level. We should test that the right entry point executes by checking for the result of running the container.